File: Sample/sample_cs/Window/Empty.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sample_cs.Window
{
	/// <summary>
	/// ウインドウを表示する。
	/// </summary>
	class Empty : ISample
	{
		public void Run()
		{
			// 初期設定を行う。
			var option = new asd.EngineOption
			{
				IsFullScreen = false
			};

			// aceを初期化する。
			asd.Engine.Initialize("Empty", 640, 480, option);

			// aceが進行可能かチェックする。
			while (asd.Engine.DoEvents())
			{
				// aceを更新する。
				asd.Engine.Update();
			}

			// aceを終了する。
			asd.Engine.Terminate();
		}
	}
}

File: Sample/BasicSample/sample_cpp/Window/Empty.cpp

#include <ace.h>

/**
	@brief	ウインドウを表示する。
*/
void Empty()
{
	// AC-Engineを初期化する。
	ace::Engine::Initialize(ace::ToAString("Empty").c_str(), 640, 480, ace::EngineOption());

	// AC-Engineのウインドウが閉じられていないか確認する。
	while (ace::Engine::DoEvents())
	{
		// AC-Engineを更新する。
		ace::Engine::Update();
	}

	// AC-Engineの終了処理をする。
	ace::Engine::Terminate();
}
